.changeset/afraid-cooks-shave.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ai-sdk/orchestration': minor
+---
+
+[Compatibility Note] Deprecate `buildAzureContentFilter()` function.
+Use `buildAzureContentSafetyFilter()` function instead and use descriptive string literals to configure the severity levels of each Azure filter category.
```

Comment:
```suggestion
Use `buildAzureContentSafetyFilter()` function instead.
```

I am not sure if this version is even any better. I will keep it simple here. We anyway have the jsdoc